src/caf/toolkit/translation.py

```diff
@@ -853,6 +853,37 @@ def pandas_vector_zone_translation(
     )
 
 
+def pandas_multi_column_translation(
+    vector: pd.DataFrame,
+    translation: pd.DataFrame,
+    from_col: str,
+    to_col: str,
+    factors_col: str,
+):
+    """
+    Translate a multicolumn dataframe between zone systems using a caf.space
```

Comment:
Initial docstring text should be in line with the opening `"""` and take up no more than 80 characters.